### ISSUE TITLE
Question: When did you stop training on your pre-trained model?

### ISSUE THREAD

**Issue by PhilipMay** [allenai/bilm-tf#135]
Hi,
when did you stop training on your pre-trained model? 
Did you stop when perplexity on validation set did increase again or before that?
Thanks
Philip

**Comment by matt-peters** [comment]
I trained for 10 epochs then stopped.  Billion word benchmark is sufficiently large that I didn't worry about overfitting, although for smaller datasets you probably want to monitor validation perplexity.  FWIW, training the ELMo model for longer reduces heldout perplexity and further improves downstream task performance.

**Comment by PhilipMay** [comment]
Ok. Thanks.
